data/json/items/generic/mail.json

```diff
@@ -38,7 +38,82 @@
     "price_postapoc": 0,
     "material": [ "paper" ],
     "symbol": "`",
-    "color": "white"
+    "color": "white",
+    "variant_type": "generic",
+    "variants": [
+      {
+        "id": "letter_home_soldier_1",
+        "name": { "str": "love letter" },
+        "description": "A written message from a soldier to their partner, carefully folded into it's envelope.  Their deployment ends soon and they can't wait to see their lover.  Reading the rest of it would be rude.",
```

Comment:
```suggestion
        "description": "A written message from a soldier to their partner, carefully folded into its envelope.  Their deployment ends soon and they can't wait to see their lover.  Reading the rest of it would be rude.",
```